dash/app/src/css/global.css

```diff
@@ -6,6 +6,10 @@
 @tailwind components;
 @tailwind utilities;
 
+body {
+  interpolate-size: allow-keywords;
```

Comment:
This is the css property I've wanted all my life. Doesn't work in all browsers yet, but on those it'll just go back to what was happening before (not animating certain transitions)

Reply:
🤩 had no idea about this, looks really neat.